compiler/rustc_errors/src/lib.rs

```diff
@@ -1995,6 +1995,17 @@ pub fn display_list_with_comma_and<T: std::fmt::Display>(v: &[T]) -> String {
     }
 }
 
+pub fn ordinalize(x: usize) -> String {
```

Comment:
I'm surprised we didn't have anything like that before 🤔

Reply:
we apparently did and then it got removed ??